src/_guides/language/analysis-options.md

```diff
@@ -301,7 +301,7 @@ Join the discussion list for linter enthusiasts:
 
 Use the following resources to learn more about static analysis in Dart:
 
-* [Strong mode](https://github.com/dart-lang/dev_compiler/blob/master/STRONG_MODE.md#strong-mode)
+* [Strong Mode Dart](/guides/language/sound-dart)
```

Comment:
All the other links aren't literal titles, so I don't think you need to change this title here. (More and more, I'm against using titles in links, since titles tend to change.)